examples/4337-passkeys/package.json

```diff
@@ -14,22 +14,22 @@
     "@account-abstraction/contracts": "0.7.0",
     "@safe-global/safe-4337": "0.3.0",
     "@safe-global/safe-contracts": "^1.4.1-build.0",
-    "@safe-global/safe-deployments": "^1.36.0",
+    "@safe-global/safe-deployments": "^1.37.0",
     "@safe-global/safe-modules-deployments": "^2.2.0",
     "@safe-global/safe-passkey": "0.2.0",
     "@web3modal/ethers": "^4.1.11",
-    "ethers": "^6.12.1",
+    "ethers": "^6.13.1",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-router-dom": "^6.23.1"
   },
   "devDependencies": {
-    "@types/react": "^18.3.2",
+    "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
-    "@vitejs/plugin-react-swc": "^3.6.0",
+    "@vitejs/plugin-react-swc": "^3.7.0",
     "react-router": "^6.23.1",
-    "typescript": "^5.4.5",
-    "vite": "^5.2.11",
+    "typescript": "^5.5.2",
+    "vite": "^5.3.1",
```

Comment:
This project refused to build on the CI due to some crazy error. I tried updating packages to see if this fixes it, but it didn't. The actual fix was related to converting the return type of ambiguous (by TS definition) call to `setInterval/setTimeout` functions to a number. It can be found in the changelog together with the explanation.

Reply:
> It can be found in the changelog together with the explanation

Which changelog are you referring to?

Reply:
> > It can be found in the changelog together with the explanation
> 
> Which changelog are you referring to?

I meant to say in "changed files" (referring to the files changed tab on GitHub)

Reply:
I found it - sorry should have resolved the comment.